app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/debugmenu/DebugMenuViewModel.kt

```diff
@@ -45,9 +47,18 @@ class DebugMenuViewModel @Inject constructor(
             DebugMenuAction.NavigateBack -> handleNavigateBack()
             DebugMenuAction.ResetFeatureFlagValues -> handleResetFeatureFlagValues()
             DebugMenuAction.ReStartOnboarding -> handleResetOnboardingStatus()
+            DebugMenuAction.ReStartOnboardingCarousel -> handleResetOnboardingCarousel()
         }
     }
 
+    private fun handleResetOnboardingCarousel() {
+        debugMenuRepository.modifyStateToShowOnboardingCarousel(
+            userStateUpdateTrigger = {
+                authRepository.hasPendingAccountAddition = true
```

Comment:
The reason this is done this way, the flow of this is only in memory in the repo, because the AuthRepository have dependencies which depend on the FeatureFlagManager (and therefore all the debug stuff under the hood) it had a circular dependency preventing a build. I did try to refactor a bit and have the DebugFFManager depend directly on the DataSource but that  introduced another dep graph issue.